tracing-api/src/main/java/com/palantir/tracing/api/TraceHttpHeaders.java

```diff
@@ -22,4 +22,16 @@
     String PARENT_SPAN_ID = "X-B3-ParentSpanId";
     String SPAN_ID = "X-B3-SpanId";
     String IS_SAMPLED = "X-B3-Sampled"; // Boolean (either “1” or “0”, can be absent)
+
+    /**
+     * Conceptually, a trace is a stack of spans. In implementation, this is actually many stacks, in many services,
```

Comment:
just to make sure this can't be misinterpreted, can we use `servers` or `hosts` or `machines` instead of `services`? I'm just aware that conjure has overloaded that term too

Reply:
donezo